src/snowflake/cli/_plugins/connection/util.py

```diff
@@ -54,6 +50,53 @@ def __init__(self, host: str | None):
         )
 
 
+class UIParameter(Enum):
+    ENABLE_REGIONLESS_REDIRECT = "UI_SNOWSIGHT_ENABLE_REGIONLESS_REDIRECT"
+    EVENT_SHARING_V2 = "ENABLE_EVENT_SHARING_V2_IN_THE_SAME_ACCOUNT"
+    ENFORCE_MANDATORY_FILTERS = (
```

Comment:
can we give this one a slightly more descriptive name? it's not obvious that this has to do with nativeapp installation

Reply:
added NA prefix